stable/storageos-operator/questions.yml

```diff
@@ -109,20 +109,26 @@ questions:
     type: string
     label: External etcd address(es)
     show_if: "cluster.kvBackend.embedded=false"
+  - variable: cluster.kvBackend.tls
+    default: false
+    type: boolean
+    description: "Enable etcd TLS"
+    label: "TLS should be configured for external etcd to protect configuration data, but it is not required."
```

Comment:
How about replacing `, but it is not required` with `(Optional)`?

Reply:
done